CMake/FindPackages.cmake

```diff
@@ -79,9 +79,6 @@ if(Boost_name)
   if(NOT "${${Boost_name}_INCLUDE_DIRS}" MATCHES "-NOTFOUND")
     include_directories(SYSTEM ${${Boost_name}_INCLUDE_DIRS})
   endif()
-  if(NOT "${${Boost_name}_INCLUDE_DIR}" MATCHES "-NOTFOUND")
```

Comment:
This file is generated by Buildyard/CMake/UseExternalDeps.cmake. You should not edit it manually.